pkg/cmd/serve/cmd.go

```diff
@@ -2,14 +2,27 @@ package serve
 
 import (
 	"context"
-	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/environments"
-	"github.com/golang/glog"
-	"github.com/spf13/cobra"
 	"os"
 	"os/signal"
+	"runtime/debug"
 	"syscall"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/environments"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
 )
 
+// showBuildInfo() returns the current Build Information using a built-in debug package and its ReadBuildInfo() function.
```

Comment:
Parentheses as part of the comment description are not specified. Just the method/function name